package/MDAnalysis/lib/pkdtree.py

```diff
@@ -127,22 +130,22 @@ def set_coords(self, coords, cutoff=None):
 
         """
         # If no cutoff distance is provided but PBC aware
```

Comment:
Move the comment with the code. Now we have an orphan comment that will confuse future contributors and maintainers.